lib/search/models/search_result.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:fluttercon/common/data/models/local/local_organiser.dart';
+import 'package:fluttercon/common/data/models/local/local_session.dart';
+import 'package:fluttercon/common/data/models/local/local_speaker.dart';
+import 'package:fluttercon/common/data/models/local/local_sponsor.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'search_result.freezed.dart';
+
+enum SearchResultType { session, speaker, sponsor, organizer }
```

Comment:
Setup all enums in the `lib/common/data/enums` folder.

Reply:
done